collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadList.java

```diff
@@ -437,7 +437,11 @@ protected void updateAdapter() {
         } else {
             ((FormDownloadListAdapter) listView.getAdapter()).notifyDataSetChanged();
         }
-
+        if(filteredFormList.size() > 0){
```

Comment:
The build is failing because of the white space here! Run `./gradlew checkstyle` or look at the build detail artifacts to see the details.

But regardless, you can simplify this by using `toggleButton.setEnabled(filteredFormList.size() > 0)`!

Reply:
Thanks @lognaturel for suggesting the changes.